src/ui/pages/GetStarted/ImportWallet/MnemonicImportView/AddressImportFlow/WalletList/WalletList.tsx

```diff
@@ -133,7 +136,7 @@ export function WalletListPresentation({
                     pad: false,
                     component: (
                       <SurfaceItemButton
-                        onClick={onLoadMore}
+                        onClick={() => onLoadMore?.()}
```

Comment:
Should we show this button on only if onLoadMore is not null?

Reply:
It's rendered based on `hasMore` condition above